README.md

```diff
@@ -1,2 +1,12 @@
-# Survey-form
-Create survey form using freecode camp guidelines
+# Survey form
+
+This repo contains files that will be used to build a survey form using freecodecamp guidelines [here](https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-survey-form).
+
```

Comment:
@ginabeki a good standard readme has the following sections:

- Requirements
- Usage/ How to run
- Contributing
You can also add support and a license but this is not really required. Check out this [blog](https://rowanmanning.com/posts/writing-a-friendly-readme/)